.github/workflows/build.yml

```diff
@@ -1,13 +1,10 @@
-name: build
+name: Builds, tests & co
 
 on:
-  pull_request:
-  push:
-    branches:
-      - master
```

Comment:
The previous version of `on:` looked better. I think this version will trigger 2 jobs everytime one update a PR. For 'push', we only care about the master branch.